packages/wdio-sumologic-reporter/src/index.ts

```diff
@@ -2,6 +2,7 @@ import got from 'got'
 import dateFormat from 'dateformat'
 import stringify from 'json-stringify-safe'
 
+import { Runner } from '@wdio/reporter/src/stats/runner'
```

Comment:
Let's expose the runner type from the package a bit better, so you can import it via:

```suggestion
import { Runner } from '@wdio/reporter'
```

Note that we don't push `src` files to NPM.